src/coreclr/jit/importer.cpp

```diff
@@ -18025,9 +18205,17 @@ void Compiler::impImportBlock(BasicBlock* block)
             }
             else if (genActualType(tree->gtType) == TYP_INT && lvaTable[tempNum].lvType == TYP_BYREF)
             {
-                // Spill clique has decided this should be "byref", but this block only pushes an "int".
-                // Insert a sign-extension to "native int" so we match the clique size.
-                verCurrentState.esStack[level].val = gtNewCastNode(TYP_I_IMPL, tree, false, TYP_I_IMPL);
+// Spill clique has decided this should be "byref", but this block only pushes an "int".
+// Insert a sign-extension to "native int" so we match the clique size.
+#ifdef TARGET_LOONGARCH64
```

Comment:
Please remove this. There should be no architecture-specific code in the importer.

Reply:
had removed.
Thanks